code/modules/security levels/security levels.dm

```diff
@@ -26,6 +26,8 @@
 				for(var/obj/machinery/firealarm/FA in world)
 					if(FA.z == ZLEVEL_STATION)
 						FA.update_icon()
+				for(var/obj/machinery/computer/shuttle/pod/pod in world)

```

Comment:
it's a computer, it might be in a computer list
it's a machine, it might be in the machines list
perhaps even define a pod_computer list, that contains only this type